### PR TITLE
removes the redundant map variables step_x & step_y

### PR DESCRIPTION
**Pull request by ghost**
[I haven't done this yet, I'm checking if travis works the way I thought it did]

### CODE REVIEW

**Comment by Exxion** [comment]
`step_x/y` don't appear in any used maps
It would be pretty obvious if they did as gliding doesn't work if they are not zero for any object on the map
Or maybe I'm misunderstanding what you're doing

**Comment by ghost** [comment]
@Exxion they do, step_x = 0 and step_y  = 0

**Comment by ghost** [comment]
but I'm just testing how to travis right now

**Comment by Exxion** [comment]
They appear but they're zero?
Wow that's stupid

**Comment by ghost** [comment]
ah, travis worked, good.

**Comment by ghost** [comment]
time to replace step_x and step_y

**Comment by ghost** [comment]
okay travis doesn't make sense to me, hm.

**Comment by ghost** [comment]
testing: this should pass the travis build.

File: maps/vgstation.dmm [hunk 1]
```diff
@@ -0,0 +1 @@
+step_x = 0
```
Comment (9600bauds) [inline maps/vgstation.dmm]: uhhh

Comment (ghost) [inline maps/vgstation.dmm]: yes I am testing travis
this is awful travis doesn't work the way I think it does

**Comment by ghost** [comment]
I don't understand how travis works

**Comment by ghost** [comment]
oh my god do you know what the reason why it was erroring is?

**Comment by ghost** [comment]
"af" = (/obj/machinery/door_control{id = 1; step_y = 32},/turf/simulated/floor,/area/engine/control)

**Comment by ghost** [comment]
ready to merge.

**Pull request merged**